app/views/devise/passwords/new.html.erb

```diff
@@ -9,8 +10,9 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Send me reset password instructions" %>
+    <%= f.submit "Send me reset password instructions", class:"this-thing" %>
```

Comment:
also the css there applies to all my buttons so Id have to call it a more general button, this-button? lol